web/src/components/ShelterInformation.tsx

```diff
@@ -43,7 +43,7 @@ const Detail = styled.div`
   display: flex;
 `
 
-const TooltipContainer = styled.div`
```

Comment:
Can you add in line 37 a box shadow for the button <3
`box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1), 0 1px 2px rgba(0, 0, 0, 0.15);`